src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4902,6 +4901,12 @@ namespace Akka.Util
         public bool IsValueCreated { get; }
         public T Value { get; }
     }
+    public sealed class FastLazy<S, T>
```

Comment:
@Rattenkrieg we could do what we did for the Helios.DedicatedThreadPool class: have the NuGet package automatically insert a new `internal FastLazy` code file into the application.

For really small packages I like that approach more than taking a dependency on another binary ("micro" packages.) Does that make sense?